bokeh/core/compat/bokeh_renderer.py

```diff
@@ -82,7 +83,7 @@ def close_figure(self, fig):
                              plot_width=self.width,
                              plot_height=self.height)
 
-                _plot.title = ""
+                _plot.title = Title()
```

Comment:
Plots have a default Title object. You can just set _plot.title.text no need to create a new Title